routes/chat.js

```diff
@@ -135,4 +136,11 @@ router.post(
   removeGroupMember
 );
 
+router.post(
+  '/group/leave',
+  validate(ChatValidation.leaveGroupMember),
+  auth.isAuthV2,
+  leaveGroupMember
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The route `/group/leave` is using HTTP POST method. However, since this operation is idempotent (leaving a group multiple times should have the same effect as leaving once), it would be more appropriate to use HTTP DELETE method. This would also align better with RESTful API design principles.
```diff
- router.post(
+ router.delete(
  '/group/leave',
  validate(ChatValidation.leaveGroupMember),
  auth.isAuthV2,
  leaveGroupMember
);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->